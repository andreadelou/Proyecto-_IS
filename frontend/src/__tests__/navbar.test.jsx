import React from 'react'
const { render } = require("@testing-library/react")
const { default: Header } = require("../components/NavBar")


describe('Test para <Navbar/>', () => {
	test('Navbar renderiza correctamente', () => {
		const { container } = render(<Header title={"Hola"} subtitle={"Mundo"} />)	
		expect(container).toMatchSnapshot()	// El snapshot es una fotografia del componente
	})
})