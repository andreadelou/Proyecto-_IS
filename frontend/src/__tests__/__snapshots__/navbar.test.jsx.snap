// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test para <Navbar/> Navbar renderiza correctamente 1`] = `
<div>
  <div
    style="display: flex; justify-content: space-between; align-items: flex-start;"
  >
    <div
      class="chakra-stack css-dhhbxq"
    >
      <h1
        class="chakra-heading css-ultdq7"
      >
        Hola
      </h1>
      <h2
        class="chakra-heading css-6orinh"
      >
        Mundo
      </h2>
    </div>
    <nav>
      <ul
        class="nav_links"
      >
        <li>
          <a
            href="#"
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Metas
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Bienestar
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Calendario
          </a>
        </li>
      </ul>
    </nav>
  </div>
</div>
`;
